.travis.yml

```diff
@@ -80,6 +81,10 @@ before_install:
       echo "library_dirs = $target/lib" >> site.cfg
       echo "include_dirs = $target/include" >> site.cfg
       echo "runtime_library_dirs = $target/lib" >> site.cfg
+      # pythran setup
+      echo '[compiler]' > ~/.pythranrc
+      echo 'CXX=g++-7' >> ~/.pythranrc
+      echo 'CC=gcc-7' >> ~/.pythranrc
```

Comment:
In the future, we may want `numpy.distutils` to get this right. Let's defer changes here till we've figured out the `build_extension` thing in `setup.py`.